src/screens/Lessons/screens/Lesson/index.js

```diff
@@ -1,20 +1,29 @@
 import React from 'react'
-import {lessonTitleText} from 'utils/text'
 import Screen from 'components/Screen'
 import LessonNextSteps from './components/LessonNextSteps'
 import LessonData from './components/LessonData'
+import WistiaVideo from './components/WistiaVideo'
 
 export default ({instructor, lesson}) => (
-  <Screen
-    title={lessonTitleText}
-    main={
-      <LessonData 
-        instructor={instructor}
-        lesson={lesson} 
-      />
-    }
-    aside={
-      <LessonNextSteps lesson={lesson} />
-    }
-  />
+  <div>
+    {lesson.wistia_id
+      ? <WistiaVideo wistiaId={lesson.wistia_id}/>
```

Comment:
That is what the `intro` prop on `Screen` is for; instead of having the div wrapper you can do this:
https://github.com/eggheadio/egghead-instructor-center/blob/master/src/screens/Lessons/screens/New/index.js#L16-L25

Which renders this top full-width piece
![image](https://cloud.githubusercontent.com/assets/5497885/23622573/72113de8-025c-11e7-8195-fec57618e99d.png)